compiler-core/src/parse/snapshots/gleam_core__parse__tests__const_invalid_bit_array_segment.snap

```diff
@@ -8,6 +8,6 @@ error: Syntax error
 2 │ const a = <<1, 2, <->>
   │                   ^^ I was not expecting this
 
-Expected one of: 
+Found "<-", expected one of: 
```

Comment:
Where do these double quotes come from? I couldn't find them in the codebase 😅 

Could you replace them all with backticks please, to match the keywords.